### PR TITLE
RDKCMF-8741 add top LICENSES.TXT file

### PR DESCRIPTION
**Pull request by marcin-hajkowski-epam**

### CODE REVIEW

File: LICENSES.TXT [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+software under the sdl-game-test directory.
+
+The sdl-game-test directory tree within this component contains some software which is
+Copyright 2021 EPAM SYSTEMS and licensed to you under the Apache 2.0 License.  The sdl-game-test
+directory contains its own license files, called LICENSE/NOTICE, which describe the licensing of the
+software under the sdl-game-test directory.
+
+The wayland-egl-test directory tree within this component contains some software which is
+licensed to you under the Unlicense. The wayland-egl-test
+directory contains its own license file, called LICENSE, which describes the licensing of the
```
Comment (dwrobel) [inline LICENSES.TXT]: Prior to that I would suggest to rename the `LICENSE` file to `UNLICENSE` as per suggestion in https://unlicense.org/:
> we recommend that you put your unlicensing statement in a file named UNLICENSE.

Plus retain the original formatting as now it's difficult to compare those two files.

File: LICENSES.TXT [hunk 2]
```diff
@@ -0,0 +1,30 @@
+The contents of this repository are subject to the RDK License Agreement.  Please refer to the RDK License Agreement signed by your company.
+
+The component may include material which is licensed under other licenses / copyrights as
+listed below. Your use of this material within the component is also subject to the terms and
+conditions of these licenses.
+
+The essos-test directory tree within this component contains some software which is
+Copyright 2017 RDK Management, Copyright 2020 Liberty Global B.V. and licensed to you
+under the Apache 2.0 License.  The essos-test directory contains its own license files,
```
Comment (dwrobel) [inline LICENSES.TXT:7]: >The essos-test directory contains its own license files,
called LICENSE/NOTICE,

There is no such a file(s).

**Review by dwrobel — CHANGES_REQUESTED** [review]

File: LICENSES.TXT (hunk 2 above)
Comment (marcin-hajkowski-epam) [reply LICENSES.TXT:7]: these files are added in this PR:
https://github.com/stagingrdkm/dac-examples-src/pull/4/

File: LICENSES.TXT (hunk 1 above)
Comment (marcin-hajkowski-epam) [reply LICENSES.TXT]: done

Comment (dwrobel) [reply LICENSES.TXT]: Thank you. Please don't forget to update `wayland-egl-test.bb` accordingly.

**Pull request merged**